lua/entities/gmod_wire_hologram.lua

```diff
@@ -56,9 +56,14 @@ if CLIENT then
 					local norm = clip.normal
 					local origin = clip.origin
 
-					if not clip.isglobal then
-						norm = self:LocalToWorld(norm) - self:GetPos()
-						origin = self:LocalToWorld(origin)
+					if clip.localentid then
+						local localent = Entity(clip.localentid)
+						if IsValid(localent) then
+							norm = localent:LocalToWorld(norm) - localent:GetPos()
+							origin = localent:LocalToWorld(origin)
+						else
+							clip.localentid = nil
```

Comment:
It's possible localent might be invalid in clientside but valid in serverside if it was just created. Probably best to not set it to nil.

Reply:
Also is Entity() capable of outputing nil? you could probably use localent:IsValid() instead

Reply:
What's the difference between the two?

Reply:
localent:IsValid() is faster